### ISSUE TITLE
Beta updates option broken

### ISSUE THREAD

**Issue by DankRafft** [lemasato/POE-Trades-Companion#248]
Whenever I deactivate "Accept BETA updates?" it gets reactivated with every start of Trades Companion. I tried directly editing the preferences.ini and set it to False but after another start of the app it's set to True again.

**Comment by lemasato** [comment]
From [1.15.BETA_1](https://github.com/lemasato/POE-Trades-Companion/releases/tag/1.15.BETA_1) changelogs:
- Fix: Would always download the beta update, even when the beta checkbox was unchecked.

Are you referring to this?
For example, trying to run 1.14.x, but it auto updates to .15.BETA_x?

**Comment by lemasato** [comment]
Whoops, misread the post. Literally, the box is being checked on app start even if you unchecked it before. I'll have to inspect more in depth.

**Comment by DankRafft** [comment]
> the box is being checked on app start even if you unchecked it before
Yes, that's what happens.

**Comment by lemasato** [comment]
One month delay, I'm sorry ._.

This is a "workaround" I made back in time to avoid a specific scenario.
Imagine the user would have "Use BETA" unchecked, and would then update manually (through GitHub) to the latest BETA.
On start, the tool would detect that the user does not have "Use BETA" checked, and automatically revert to the latest stable.

For this specific reason, I made it work this way: If the version is a BETA, the UseBeta is set to be True. 
Though, I have a better solution in mind. Simply comparing pre-update version and post-update version, then tick/untick the box accordingly.
